pkgs/applications/networking/browsers/firefox/wrapper.nix

```
@@ -102,14 +102,11 @@ let
 
       nameArray = builtins.map(a: a.name) (lib.optionals usesNixExtensions nixExtensions);
 
-      requiresSigning = browser ? MOZ_REQUIRE_SIGNING
-                     -> toString browser.MOZ_REQUIRE_SIGNING != "";
-
       # Check that every extension has a unqiue .name attribute
       # and an extid attribute
       extensions = if nameArray != (lib.unique nameArray) then
         throw "Firefox addon name needs to be unique"
-      else if requiresSigning && !lib.hasSuffix "esr" browser.name then
+      else if browser.requireSigning && !lib.hasSuffix "esr" browser.name then
         throw "Nix addons are only supported without signature enforcement (eg. Firefox ESR)"
```

This won't hold true anymore with the end of life of the 102 ESR series.

```suggestion
        throw "Nix addons are only supported without signature enforcement (Librewolf or Firefox Developer Edition)"
```

---

Oh they're finally changing that? Huh. We should probably add some additional checks for that then.

---

Updated the text, added `eg.` in front of your wording for consistency. For my own reference, was this change announced anywhere? I checked a few changelogs for ESR and didn't see any mention of addon signing changes.